apps/projects/app/components/Shared/FilterBar/FilterBar.js

```diff
@@ -112,11 +143,128 @@ const SortPopover = ({ visible, opener, setSortMenuVisible, sortBy, updateSortBy
 SortPopover.propTypes = {
   visible: PropTypes.bool.isRequired,
   opener: PropTypes.object,
-  setSortMenuVisible: PropTypes.func.isRequired,
+  setVisible: PropTypes.func.isRequired,
   sortBy: PropTypes.string.isRequired,
   updateSortBy: PropTypes.func.isRequired,
 }
 
+const ActionsPopover = ({ visible, setVisible, openerRef, selectedIssues, issuesFiltered, deselectAllIssues }) => {
+  const { curateIssues, allocateBounty } = usePanelManagement()
+
+  return (
+    <Popover
+      visible={visible}
+      opener={openerRef.current}
+      onClose={() => setVisible(false)}
+      placement="bottom-end"
+      css={`
+        display: flex;
+        flex-direction: column;
+        padding: 10px;
+      `}
+    >
+      <FilterMenuItem
+        key="1"
+        onClick={() => {
+          curateIssues(selectedIssues, issuesFiltered)
+          deselectAllIssues()
+          setVisible(false)
+        }}
+      >
+        <IconFilter />
+        <ActionLabel>Curate Issues</ActionLabel>
+      </FilterMenuItem>
+      <FilterMenuItem
+        key="2"
+        onClick={() => {
+          allocateBounty(selectedIssues)
+          deselectAllIssues()
+          setVisible(false)
+        }}
+      >
+        <IconCoins />
+        <ActionLabel>Fund Issues</ActionLabel>
+      </FilterMenuItem>
+    </Popover>
+  )
+}
+ActionsPopover.propTypes = {
+  visible: PropTypes.bool.isRequired,
+  setVisible: PropTypes.func.isRequired,
+  openerRef: PropTypes.object.isRequired,
+  selectedIssues: PropTypes.arrayOf(issueShape).isRequired,
+  issuesFiltered: PropTypes.arrayOf(issueShape).isRequired,
+  deselectAllIssues: PropTypes.func.isRequired,
+}
+
+const Actions = ({ onClick, openerRef, visible, setVisible, selectedIssues, issuesFiltered, deselectAllIssues,  }) => {
+  const { layoutName } = useLayout()
+  const theme = useTheme()
+
+  const actionsButtonBg = () =>
+    'background-color: ' + (!selectedIssues.length ? `${theme.background}` : `${theme.surface}`)
+
+  return (
+    <React.Fragment>
+      {layoutName === 'large' ? (
+        <Button
+          css={actionsButtonBg()}
+          onClick={onClick}
+          ref={openerRef}
+        >
+          <IconGrid />
+          <Text css="margin: 0 8px;">Actions</Text>
+          <IconArrowDown />
+        </Button>
+      ) : (
+        <Button
+          css={actionsButtonBg()}
+          onClick={onClick}
+          ref={openerRef}
+          icon={<IconGrid />}
+          display="icon"
+          label="Actions Menu"
+        />
+      )}
+      <ActionsPopover
+        openerRef={openerRef}
+        onClick={onClick}
```

Comment:
It did work, although in confusing manner: without an selected issues it would have a slighly darker background to mark that it's disabled, without being fully disabled (just not showing anything useful upon clicking). So, it was agreed on #design that when no issues are selected the button should not be even there - it appears when an issue gets selected, and so users attention is informed that new UI action becomes avaiable.